config/capi/_ytt_lib/capi-k8s-release/templates/api_server_deployment.yml

```diff
@@ -77,11 +77,34 @@ spec:
           #@ if/end data.values.ccdb.ca_cert:
           - name: database-ca-cert
             mountPath: /config/database/certs
+        - name: package-image-uploader
```

Comment:
This work is related to https://github.com/cloudfoundry/cf-for-k8s/issues/409. It's still in flight and isn't completely finished yet, we'll probably want to hold off on merging this (and autobumping capi-k8s-release) until that's complete as well.

Reply:
Discussing this on Slack: https://cloudfoundry.slack.com/archives/C017LDM6KTQ/p1599088788032100